buildroot-external/patches/occu/0170-WebUI-Add-HmIPKeyTransceiverCentralLinkConfiguration/occu/WebUI/www/webui/webui.js

```diff
@@ -13663,13 +13664,8 @@ ChannelConfigDialog = Singleton.create({
     new YesNoDialog(translateKey('dialogSafetyCheck'), translateKey('dialogQuestionRemoveLink').replace('%s', addr+' - ZENTRALE'), function(result) {
       if (result == YesNoDialog.RESULT_YES) {
 		  if (channeltype === "KEY_TRANSCEIVER") {
-            ResetPostString();
-            AddParam(document.getElementById('global_sid'));
-            poststr += "&cmd=removeLink";
-            poststr += "&iface=HmIP-RF";
-            poststr += "&sender_address="+addr;
-            poststr += "&receiver_address=CENTRAL_DEVICE:63" ;
-            SendRequest('ic_ifacecmd.cgi');
+		    //reportValueUsage for HmIP ignores the valueId, so it doesn't matter what you put in there
+            homematic("Interface.reportValueUsage", {"interface": "HmIP-RF", "address": addr, "valueId": "0", "refCounter":"0"});  
```

Comment:
Ist das ein "Problem" das man bei HmIP die `valueID` nicht angeben kann/muss?

Reply:
Kann ich nicht beurteilen, da ich bei der Problematik hier nur derjenige bin, der das Gewünschte in Code umsetzt.

Ich hatte selbst noch nie Berührungspunkte mit dem `reportValueUsage` und jetzt im Laufe des Issues/PR hier erst gelernt, dass bei BidCos/Wired mittles `valueID` unterschieden wird, ob SHORT_PRESS und/oder LONG_PRESS an die Zentrale gesendet werden sollen (_Der Parameter `valueID` = `SHORT_PRESS` bzw. `LONG_PRESS`._)

Wenn z.B. jemand in einem Programm "WENN Tastendruck lang..." als Trigger wählt, dann wird auch nur für den langen Tastendruck ein Event an die Zentrale gesendet.
Ist ein kurzer Tastendruck mit einem Aktor direktverknüpft, würde sodann der kurze Tastendruck auch nur an den Aktor und nicht die Zentrale gesendet werden.

Bei HmIP gibt es diese Differenzierung offensichtlich nicht.